packages/webcomponents/src/components/business-form/business-form-stepped.tsx

```diff
@@ -26,25 +18,21 @@ const componentStepMapping = {
 export class BusinessFormStepped {
   @Prop() authToken: string;
   @Prop() accountId: string;
-  @Prop() businessId?: string;
+  @Prop() businessId: string;
   @Prop() testMode: boolean = false;
-  @Prop() hideErrors?: boolean;
+  @Prop() hideErrors?: boolean = false;
   @State() isLoading: boolean = false;
   @State() currentStep: number = 0;
   @State() totalSteps: number = 4;
   @State() serverError: boolean = false;
   @State() errorMessage: string = '';
 
-  get submitDisabled() {
-    return !this.authToken || this.isLoading || this.serverError;
-  }
-
-  get nextDisabled() {
-    return !this.authToken || this.isLoading || this.serverError;
+  get disabledState() {
+    return !this.authToken || !this.businessId || this.isLoading || this.serverError;
```

Comment:
I don't think this logic works because if a user attempts to save something that results in a server error and they correct it, they will not be able to try submitting again. I also think that we should leave `!this.authToken` and `!this.businessId` out and just let the console errors do the work to lower code complexity